### PR TITLE
Update code snippets to pull from sample code for Blobs

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/samples/Sample01a_HelloWorld.cs [hunk 1]
```diff
@@ -129,68 +133,93 @@ public void DownloadImage()
...
                 Assert.Fail($"Unexpected error: {ex}");
             }
+        }
 
-            // Clean up after the test when we're finished
-            container.Delete();
+        /// <summary>
+        /// Authenticate with <see cref="DefaultAzureCredential"/>.
+        /// </summary>
+        [Test]
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/samples/Sample01a_HelloWorld.cs]: This probably shouldn't be a test

File: sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs [hunk 2]
```diff
@@ -137,38 +149,45 @@ public void FineGrainedBatching()
...
-                // An aggregate exception is thrown for all the indivudal failures
+
+            //@@ catch (AggregateException ex)
+            //@@ {
+                // An aggregate exception is thrown for all the individual failures
+                // Check ex.InnerExceptions for RequestFailedException instances
+            //@@ }
+            #endregion
+
+            catch (AggregateException ex) {
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs]: Curly on the next line

File: sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs [hunk 3]
```diff
@@ -137,38 +149,45 @@ public void FineGrainedBatching()
...
+            BlobBatchClient batch = service.GetBlobBatchClient();
+            try
+            {
                 batch.DeleteBlobs(new Uri[] { valid.Uri, invalid.Uri });
             }
-            catch (AggregateException ex)
-            {
-                // An aggregate exception is thrown for all the indivudal failures
+
+            //@@ catch (AggregateException ex)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs]: Why are we commenting this out?

Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs]: Let's just remove the verification code

**Review by tg-msft — APPROVED** [review]

**Comment by tg-msft** [comment]
Please also update the Batch changelog/breaking changes to note the rename of Batch to Batching "for consistency with other platforms"

File: sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs (hunk 3 above)
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs]: But then why have it as a test?

Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs/samples/Sample03a_Batching.cs]: We'll still know if the exception ever changes.  I care more about the readability right now and this is hard to read for a sample.  We can make the test better after releasing.

File: sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt [hunk 4]
```diff
@@ -1,5 +1,34 @@
 Breaking Changes
 ================
+12.0.0 (2019-11)
+--------------------------
+- Make ContentLanguage and ContentEncoding properties strings everywhere as opposed to being arrays or IEnumerable<string> in some places.
```
[the remark this answers is not in the document]
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt]: Don't have a strong opinion on this, thought it looks like we were at least intending to list every breaking change out in prior releases.

**Comment by tg-msft** [comment]
Also remove edit your READMEs to remove the version from the "install" section so it's just `dotnet add package Azure.Storage.Blobs`

File: sdk/storage/Azure.Storage.Blobs.Batch/README.md [hunk 5]
```diff
@@ -154,7 +154,7 @@ try
     batch.DeleteBlobs(new Uri[] { valid.Uri, invalid.Uri });
 }
 
-catch (AggregateException ex)
+catch (AggregateException)
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs.Batch/README.md]: Nit - leave the `ex` cause you talk about it two lines down.

File: sdk/storage/Azure.Storage.Blobs/README.md [hunk 6]
```diff
@@ -127,11 +123,7 @@ We fully support both synchronous and asynchronous APIs.
 string downloadPath = "hello.jpg";
 
 // Download the public blob at https://aka.ms/bloburl
-BlobDownloadInfo download = await new BlobClient(new Uri("https://aka.ms/bloburl")).DownloadAsync();
-using (FileStream file = File.OpenWrite(downloadPath))
-{
-    await download.Content.CopyToAsync(file);
-}
+await new BlobClient(new Uri("https://aka.ms/bloburl")).DownloadToAsync(downloadPath);
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/README.md]: Wait - why is this here twice?  Did we get the snippets wrong?  Should this be the enumerating blobs sample?

**Review by tg-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs/BreakingChanges.txt]: I don't think we've got them all.  Let's change it to

```
- Renamed a a number of operations and models to better align with other client
  libraries and the .NET Framework Design Guidelines
```

File: sdk/storage/Azure.Storage.Blobs/README.md (hunk 6 above)
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Blobs/README.md]: This is the Async snippet

**Review by amnguye — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs.Batch/README.md (hunk 5 above)
Comment (JoshLove-msft) [reply sdk/storage/Azure.Storage.Blobs.Batch/README.md]: This causes a compiler error that would need to be suppressed. Suppressing it would pull the suppression comment into the README.

File: sdk/storage/Azure.Storage.Blobs/README.md (hunk 6 above)
Comment (tg-msft) [reply sdk/storage/Azure.Storage.Blobs/README.md]: Ah, sorry.  The diff said it was part of enumeration.  My bad.
